HopsanGUI/MainWindow.cpp

```diff
@@ -577,7 +577,7 @@ void MainWindow::createActions()
     mpDisableUndoAction->setCheckable(true);
     mpDisableUndoAction->setChecked(false);
 
-    mpOpenSystemParametersAction = new QAction(QIcon(QString(ICONPATH) + "Hopsan-SystemParameter.png"), tr("&System Parameters"), this);
+    mpOpenSystemParametersAction = new QAction(QIcon(QString(ICONPATH) + "svg/Hopsan-GlobalParameter.svg"), tr("&System Parameters"), this);
```

Comment:
I know this is only the file name, but it suggests that these parameters are global, which they are not. At least in the way  it works right now. Buy I guess it makes sense since the icon has changed.